From: Tomas Henzl <thenzl@redhat.com>
Date: Sun, 29 Aug 2010 15:50:00 -0400
Subject: [block] cciss: rename cciss_sector_size
Message-id: <1283097002-3341-62-git-send-email-thenzl@redhat.com>
Patchwork-id: 27895
O-Subject: [RHEL6 PATCH 61/63] cciss: rename cciss_sector_size
Bugzilla: 568830
RH-Acked-by: Neil Horman <nhorman@redhat.com>

rename cciss_sector_size field to cciss_max_sectors, as that is what it is.

diff --git a/drivers/block/cciss.c b/drivers/block/cciss.c
index a6c08db..920585c 100644
--- a/drivers/block/cciss.c
+++ b/drivers/block/cciss.c
@@ -362,7 +362,7 @@ static void cciss_seq_show_header(struct seq_file *seq)
 		h->firm_ver[0], h->firm_ver[1], h->firm_ver[2],
 		h->firm_ver[3], (unsigned int)h->intr[SIMPLE_MODE_INT],
 		h->num_luns,
-		h->cciss_sector_size,
+		h->cciss_max_sectors,
 		h->Qdepth, h->commands_outstanding,
 		h->maxQsinceinit, h->max_outstanding, h->maxSG);
 
@@ -1851,7 +1851,7 @@ static int cciss_add_disk(ctlr_info_t *h, struct gendisk *disk,
 	/* This is a limit in the driver and could be eliminated. */
 	blk_queue_max_phys_segments(disk->queue, h->maxsgentries);
 
-	blk_queue_max_sectors(disk->queue, h->cciss_sector_size);
+	blk_queue_max_sectors(disk->queue, h->cciss_max_sectors);
 
 	blk_queue_softirq_done(disk->queue, cciss_softirq_done);
 
@@ -4189,7 +4189,7 @@ static int __devinit cciss_init_one(struct pci_dev *pdev,
 	kfree(inq_buff);
 
 	cciss_procinit(i);
-	hba[i]->cciss_sector_size = 8192;
+	hba[i]->cciss_max_sectors = 8192;
 	hba[i]->busy_initializing = 0;
 	rebuild_lun_table(hba[i], 1, 0);
 
diff --git a/drivers/block/cciss.h b/drivers/block/cciss.h
index e35bbac..5558cce 100644
--- a/drivers/block/cciss.h
+++ b/drivers/block/cciss.h
@@ -95,7 +95,7 @@ struct ctlr_info
 	unsigned int msi_vector;
 	BYTE	cciss_read;
 	BYTE	cciss_write;
-	int	cciss_sector_size;	
+	int	cciss_max_sectors;	
 
 	// information about each logical volume
 	drive_info_struct *drv[CISS_MAX_LUN];
